foundation-model-stack/fms-hf-tuning
# feat: Configuration to set logging level for trigger log

### Description of the change

The log message printed when an action is taken by the trainer controller might require different logging levels for different use-cases.

The trainer controller has a configuration section where we could specify the trigger log level.

### Related issue number

"Closes #242" 

### How to verify the PR

Use this [example](examples/trainercontroller_configs/epoch-level-training-loss-below-threshold-log-config.yaml) to verify the trigger log-level configuration does indeed emit the loss at that level.

<img width="1725" alt="image" src="https://github.com/user-attachments/assets/8198efcf-5a7a-4eac-8a7f-0fbd7bbeb759">


### Was the PR tested
